src/main/java/uk/gov/hmcts/reform/em/stitching/pdf/PDFMerger.java

```diff
@@ -169,9 +181,40 @@ private void addDocument(SortableBundleItem item) throws IOException {
             }
             pdfOutline.closeParentItem();
             currentPageNumber += newDoc.getNumberOfPages();
+
             newDoc.close();
         }
 
+        private void addDocumentImage(PDDocument document) throws IOException {
```

Comment:
I don't think this method should be in here - there's already too much stuff.

Move it to it's own class, then you can call it from DocumentTaskItemProcessor after the merge is done